docs/manage/config-policies.rst

```diff
@@ -0,0 +1,263 @@
+.. _config-policies:
+
+#######################
+ Config Policies Guide
+#######################
+
+Config Policies allow administrators to set limits on how users can define workloads (e.g.,
+experiments, notebooks, tensorboards, shells, and commands). Administrators can include any
+parameter referenced in the :ref:`experiment configuration reference <experiment-config-reference>`.
+
+.. include:: ../_shared/attn-enterprise-edition.txt
+
+Config Policies allow admins to define two types of configurations:
+
+-  **Invariant Configs**: Settings applied to all workloads within a scope (global or workspace).
+-  **Constraints**: Restrictions that prevent users from exceeding resource limits.
+
+These policies are essential for managing task priorities, resource allocation, and setting specific
+configurations across the organization. Global policies override workspace policies, and both
+override user-submitted configurations.
+
+**************
+ Key Features
+**************
+
+-  Apply policies via WebUI or CLI
+-  Define limits for resource usage, environment settings, and more
+-  Apply policies at different levels (cluster-wide or workspace)
+-  Override capabilities for specific user roles or groups
+-  Set different priority limits for experiments and NTSC (notebooks, TensorBoards, shells, and
+   commands) tasks
+-  Allow priority overrides within policy constraints
+
+******************
+ Setting Policies
+******************
+
+Administrators can set Config Policies at either the cluster or workspace levels through the WebUI
+or CLI.
+
+WebUI
+=====
+
+Administrators can set Config Policies at both the cluster and workspace levels.
+
+.. tabs::
+
+   .. tab::
+
+      Cluster Config Policy
+
+      To set configuration policies at the cluster level:
+
+      #. Sign in to the Determined WebUI as a cluster administrator.
+      #. Navigate to **Config Policies**.
+      #. Choose **Experiments** or **Tasks** to display the editable configuration file.
+      #. Define the policies, then click **Apply**.
+      #. Confirm you want to apply the policies.
+
+   .. tab::
+
+      Workspace Config Policy
+
+      To set configuration policies at the workspace level:
+
+      #. Sign in to the Determined WebUI as a cluster or workspace administrator.
+      #. From your workspace, navigate to the **Config Policies** tab.
+      #. Choose **Experiments** or **Tasks** to display the editable configuration file.
+      #. Define the policies, then click **Apply**.
+      #. Confirm you want to apply the policies.
+
+CLI
+===
+
+Use the following commands to manage Task Config Policies via CLI:
+
+.. code:: bash
+
+   # List existing policies
+   det policy list
+
+   # Create a new policy
+   det policy create <policy_name> --config <policy_config_file.yaml>
```

Comment:
```suggestion
   # Create a new policy
   det config-policies set <workload_type> --workspace --config-file <policy_config_file>
   
 where <workload_type> can be either `experiment` or `ntsc` and <policy_config_file> is the absolute or relative path the YAML or JSON file containing your config policies.
```

Reply:
On the note of the `<workload_type>` being `experiment` or `ntsc` I brought up using "NTSC" instead of "Tasks" in the webUI during a demo a couple of weeks ago, and someone (I think Tim?) mentioned that customers aren't very familiar with the NTSC terminology /categorization. If that's the case, it could be worth changing `ntsc` to `tasks` in the CLI for consistency (if time permits)!

Reply:
@amandavialva01 changes applied but, keeping this open in case you want to retain your note about the CLI. Do we need to share this with the CLI engineer?

Reply:
Yes! Saloni was the CLI developer, but she is currently on rotation on a different team!  We currently have one [ticket](https://hpe-aiatscale.atlassian.net/jira/software/c/projects/CM/boards/73?assignee=unassigned&assignee=712020%3A44fd218d-5992-489b-810a-0e9cb2330674&selectedIssue=CM-576) left for CLI work that I picked up. 
But we have been tackling high priority fixes/work before making other changes. With that being said, I do think the only remaining "bug" fix is in review and a change like this could be made in the CLI ticket I linked (CM-576).
WDYT Kristine? Should we make this change after comments from the demo two weeks ago?